packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -30,6 +30,17 @@ let _defaultMode: ResponsiveMode | undefined;
  */
 let _lastMode: ResponsiveMode | undefined;
 
+/**
+ * Initializes the responsive mode to the current window size. This can be used to avoid
+ * a re-render during first component mount since the window would otherwise not be measures
+ * until after mounting.
+ */
+export function initializeResponsiveMode(): void {
+  if (typeof window !== 'undefined') {
```

Comment:
Should this use the getWindow utility?

Reply:
That requires an element to be provided, which the caller won't typically have. But I'll modify this function to accept one optionally.